src/test/cls_2pc_queue/test_cls_2pc_queue.cc

```diff
@@ -39,7 +51,7 @@ class TestCls2PCQueue : public ::testing::Test {
 
 TEST_F(TestCls2PCQueue, GetCapacity)
 {
-  const std::string queue_name = "my-queue";
+  const std::string queue_name = generate_q_name();
```

Comment:
i like to use the function name as the oid in tests, ie:
```c++
const std::string oid = __PRETTY_FUNCTION__;
```
this has the same benefits as randomized oids, but also allows you to associate log output of these object names with the test case that's operating on them